Vue/README.md

```diff
@@ -0,0 +1,19 @@
+# {{ name }}
+
+> {{ description }}
+
+## Build Setup
+
+```bash
+# install dependencies
+npm install
+
+# serve with hot reload at localhost:8080
+npm run dev
+
+# build for production with minification
+npm run build
+```
+
+For detailed explanation on how things work, consult the
+[docs for vue-loader](http://vuejs.github.io/vue-loader).
```

Comment:
`vue-loader` even used?

Reply:
- [x] Nope, it's not used at the moment, I've fixed this in 89c504a